t/plugin/consumer-restriction.t

```diff
@@ -475,7 +474,7 @@ Authorization: Basic amFjazIwMjA6MTIzNDU2
 
 
 
-=== TEST 21: remove consumer-restriction
```

Comment:
Why change this test?

Reply:
I bump the remove restriction to test 31 due to the tests i added and to remains consistent .